### ISSUE TITLE
CLI output log to stdout/stderr

### ISSUE THREAD

**Issue by pixil98** [rmcrackan/Libation#626]
**Is your feature request related to a problem? Please describe.**
I'm trying to debug some issues where my weekly job that downloads new books is getting errors and writing out m4b files that are only 10MB. Since this is running in a docker image, it's hard to debug because the logging all to a file that I don't have access to after the docker image finishes running.

**Describe the solution you'd like**
It would be nice if there was an option to for the cli to write it's logs to stdout and stderr. This is the expectation with a docker image and would allow those logs to be scraped and review easily.

**Describe alternatives you've considered**
There are some ways to trick a program into writing to stdout, depending on how it interacts with log files. I haven't actually tried using them yet.

**Comment by rmcrackan** [comment]
I haven't tried this, but you might be able to just add this into your Settings.json in the WriteTo array

```
      ,{
        "Name": "Console",
        "Args": {
          "outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] (at {Caller}) {Message:lj}{NewLine}{Exception} {Properties:j}"
        }
      }
```

**Comment by rmcrackan** [comment]
Included in [Libation v10.4](https://github.com/rmcrackan/Libation/releases/latest)
